src/SignalR.Orleans/AllMessage.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.SignalR.Protocol;
+using Orleans.Concurrency;
+
+namespace SignalR.Orleans
+{
+    [Immutable, GenerateSerializer]
+    public sealed record AllMessage(Immutable<InvocationMessage> Message, IReadOnlyList<string>? ExcludedIds = null);
```

Comment:
Just FYI, you can use the `[Immutable]` attribute here as well and simplify the consumption by removing the `.Value` call everywhere.